components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java

```diff
@@ -774,4 +773,23 @@ private static boolean isFormatCorrect(String regularExpression, String domainNa
         Matcher m2 = p2.matcher(domainName);
         return m2.matches();
     }
+
+    /**
+     * Triggers post tenant delete for TenantMgtListener.
+     *
+     * @param tenantId int Tenant id.
+     * @param tenantUuid String Tenant unique identifier.
+     * @param adminUserUuid String Tenant admin user unique identifier.
+     * @throws StratosException  If trigger failed.
```

Comment:
Foramatting issue

Reply:
fixed in https://github.com/wso2/carbon-multitenancy/pull/219